src/transformers/pipelines/image_segmentation.py

```diff
@@ -79,30 +80,35 @@ def __call__(self, *args, **kwargs) -> Union[Predictions, List[Prediction]]:
 
                 The pipeline accepts either a single image or a batch of images. Images in a batch must all be in the
                 same format: all as HTTP(S) links, all as local paths, or all as PIL images.
+            task (`str`, defaults to `semantic`):
+                Segmentation task to be performed, choose [`semantic`, `instance` and `panoptic`] depending on model
+                capabilities.
             threshold (`float`, *optional*, defaults to 0.9):
-                The probability necessary to make a prediction.
-            mask_threshold (`float`, *optional*, defaults to 0.5):
-                Threshold to use when turning the predicted masks into binary values.
+                Probability threshold to filter out predicted masks.
+            overlap_mask_area_threshold (`float`, *optional*, defaults to 0.5):
+                Mask overlap threshold to eliminate small, disconnected segments.
 
         Return:
             A dictionary or a list of dictionaries containing the result. If the input is a single image, will return a
             list of dictionaries, if the input is a list of several images, will return a list of list of dictionaries
             corresponding to each image.
 
-            The dictionaries contain the following keys:
+            The dictionaries contain the mask, label and score (where applicable) of each detected object and contains
+            the following keys:
 
             - **label** (`str`) -- The class label identified by the model.
-            - **mask** (`PIL.Image`) -- Pil Image with size (heigth, width) of the original image. Pixel values in the
-              image are in the range 0-255. 0 means the pixel is *not* part of the *label*, 255 means it definitely is.
+            - **mask** (`PIL.Image`) -- A binary mask of the detected object as a Pil Image of shape (heigth, width) of
```

Comment:
If you call `image.size` on a PIL image, it's in (width, height) format.
```suggestion
            - **mask** (`PIL.Image`) -- A binary mask of the detected object as a PIL Image of shape (width, height) of
```